### ISSUE TITLE
JS DOM design icons problem

### ISSUE THREAD

**Issue by krssclaire** [krssclaire/todo-list#1]
### Problem

Some icons used in .sidebar and .list divs are not visible, while other icons are.

In particular the icons that are not visible are:
* In .sidebar div:
  - homeIcon before the Task list;
  - starEmptyIcon before the Important list;
  - trashICon before the Completed List.
* In .list div:
  - notesIcon that is supposed to be after the line divider |

**Notes**:	
* The notesIcon is also used in .details div, but here it works well, while in .list it doesn't.
*  Same with the homeIcon used in .list, here it's displayed while in .sidebar it isn't.

### Remarks

When checking the DOM in browser developer tool, it seems like the missing icons aren't added in the DOM at all despite the code does so.

### Notes

1. The temporary design  structure made with JS DOM uses modules and module patterns: index.js puts together the UI components created in different modules.
   Each exportable module (Header, Sidebar, List and Details) creates elements, then sets attributes and text content and in the end it appends the created elements in the correct order to create the hierarchy in the DOM.

2. This project currently features an Icon module which is a collection of all icons-objects that are used in the project so that they can be accessed very easily every time they're needed.
   Each module (Header, Sidebar, List container and Details bar) that will need icons, will import them directly from the Icons module.
   Long story short, this module file (src/modules/icons-collection.js) imports images files, then creates img elements and assigns the correct src, and finally returns images as objects to be inserted in the DOM.

**Comment by krssclaire** [comment]
### Explanation

The fact that the project has a module made of icons that can be recycled, caused the icons displaying problem.
Basically, we cannot assign a child node multiple parents node, so the child's parent node is the last one defined.
Link: [here](https://stackoverflow.com/questions/30667088/why-isnt-possible-to-append-a-same-child-in-two-dom-element)


### Solution

To make the design work, it could be useful to clone the node using the **cloneNode()** method, or create a function that generates a new node whenever it's needed.

**Note**: these solutions haven't been found by me, but they've been suggested by a TOP student in the official Discord channel.
